src/bin/pgcopydb/pgcmd.h

```diff
@@ -191,6 +191,7 @@ typedef struct RestoreOptions
 	bool noACL;
 	bool noTableSpaces;
 	int jobs;
+	char section[BUFSIZE];
```

Comment:
Style issue: either have an enum datatype for internal option management, or just add a `const char *section` argument to `pg_restore_db`.

Reply:
Ok. Can you point me to an example in the sources, please?

Reply:
The structure `RestoreOptions` itself is a good example, along with code such as the following. As you can see we do not store the string "--no-owner" in the structure, we store the C boolean information that the option noOwner has been selected. This is expanded to the command line argument, as a string, as needed, when preparing to command line.

``` c
	if (options.noOwner)
	{
		args[argsIndex++] = "--no-owner";
	}
```

Reply:
gotcha! 👍

Reply:
done